src/modules/groups/groups.js

```diff
@@ -24,65 +22,29 @@ const getAccessToken = async () => {
         }
       }
     );
-    return { access_token };
+    return { accessToken: data?.access_token };
   } catch (error) {
     return { error };
   }
 };
 
-const accessToken = async (req, res) => {
-  const { access_token, error } = (await getAccessToken()) || {};
+router.get('/token', async (req, res) => {
+  const { accessToken, error } = (await getAccessToken()) || {};
   if (error) {
     return res.status(400).send({
       error
     });
   }
   return res.status(200).send({
-    access_token
+    accessToken
   });
-};
-
-const validateJWT = async (req, res) => {
-  const { jwt } = req.body || {};
-  const { tenantId, clientId } = configurations[process.env.NODE_ENV];
-
-  if (!jwt) {
-    return res.status(400).send({
-      message: 'JWT missing from body'
-    });
-  }
-
-  if (!tenantId || !clientId) {
-    return res.status(400).send({
-      message: 'tenantId and clientId are needed for JWT validation'
-    });
-  }
-
-  try {
-    const decodedToken = await validate(jwt, {
-      tenantId: tenantId,
-      audience: clientId,
-      applicationId: 'f7fdabfa-274a-4401-852d-b448a62f70d6'
-      // scopes: 'YOUR_SCOPES', // for example ["User.Read"]
-    });
-
-    res.status(200).send({
-      decodedToken
-    });
-
-    // DO WHATEVER YOU WANT WITH YOUR DECODED TOKEN
-  } catch (error) {
-    // ALL ERRORS GONNA SHOW HERE AS A STRING VALUE
-    return res.status(400).send({
-      message: error
-    });
-  }
-};
+});
 
 router.get('/all/:type', async (req, res) => {
   const { type } = req.params;
-  const { MS_GRAPH_HOST_NAME, WRITERS_GROUP_ID, EDITOR_GROUP_ID } = process.env;
-  const { access_token } = (await getAccessToken()) || {};
+  const { MS_GRAPH_HOST_NAME, WRITERS_GROUP_ID, EDITOR_GROUP_ID, SIZING_GROUP_ID, ADMIN_GROUP_ID } =
+    process.env;
+  const { accessToken } = (await getAccessToken()) || {};
```

Comment:
to avoid this initialization, 
update your getAccessToken function to return access token directly, remove object return

Reply:
handle in getAccessToken function